src/locales/en/messages.po

```diff
@@ -0,0 +1,356 @@
+msgid ""
```

Comment:
any plan to get the rest of the existing translations in here? in a way that isn't too labor intensive? or we can just get a new pass of translators after everything has been wrapped in `Trans`

Reply:
Current plan:
-Wrap all English text in app around <Trans>
-Review and get approval on all keys
-Generate new {locale}.po files for languages we support
-Auto-fill {locale}.po with previous translations (hoping to automate with a script)
-Notify translators to translate remaining keys